client/src/pages/contact.tsx

```diff
@@ -12,18 +12,17 @@ interface ContactPageProps {
 
 const ContactPage = ({location}: ContactPageProps) => {
   const generalemail = 'screeningtool.feedback@usds.gov';
-  const techemail = 'screeningtool.support@usds.gov';
 
   return (
     <Layout location={location}>
 
       <J40MainGridContainer fullWidth={true}>
-        <AlertWrapper />
+        <AlertWrapper/>
       </J40MainGridContainer>
 
       <J40MainGridContainer>
-        <Grid row><Grid col>
-          <section className={'usa-prose'}>
+        <section className={'usa-prose'}>
```

Comment:
On some of the pages I think I remember seeing the pattern of Container > row > col > section. Is there something special here that we have to break from that pattern?

Reply:
Yeah. I'm not sure the correct approach here.  HTML spec says: 

> The <section> HTML element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it. Sections should always have a heading, with very few exceptions.

So, it's kind of like a higher level `<h#/><p/>` part of the document?

It sounds like it's a fallback for html5 if the content doesn't fit nicely into `<article>`, `<aside>`, `<main>`, `<nav>`, etc.

One challenge of using Trussworks react library is that I'm not aware of if/where it uses html5 tags.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Not sure how close we're sticking to html5 tags in the rest of our page output?

Reply:
Removed `<section>` since it's already in a `<main>`